TESTING.md

```diff
@@ -1,22 +1,24 @@
 # Testing 
-End-to-end tests for CHT-Sync can be found under `tests/`. In order to run these end-to-end tests locally, follow the steps below:
+End-to-end tests for CHT Sync can be found under `tests/`. 
 
-1. Create an `.env` file with the placeholder values from `env.template`.
+Pre-requisites:
+* `Node.js` version > 20
 
-2. Install the dependencies via:
+In order to run these end-to-end tests locally, follow the steps below:
+
+1. Install the dependencies via:
 
 ```sh
 npm install
 ```
 
-3. Run the Docker containers locally:
+2. Install the `couch2pg` dependencies via:
```

Comment:
Do you think we should make this a post install hook?

Reply:
Good point! I don't see any drawback, and removing a manual step is a small win. I will add another commit shortly to address this.

Reply:
Done